source/Public/Utils/Invoke-FabricAPIRequest_duplicate.ps1

```diff
@@ -119,7 +119,10 @@ function Invoke-FabricAPIRequest_duplicate {
                     } else {
                         $results += $response
                     }
-                    $continuationToken = $response.PSObject.Properties.Match("continuationToken") ? $response.continuationToken : $null
+                    $continuationToken = $null
+                    if ($response.PSObject.Properties.Match("continuationToken")) {
+                        $continuationToken = $response.continuationToken
+                    }
```

Comment:
I think I'd probably do this as an if \else - would say this is more standard PowerShell syntax

```suggestion
                    if ($response.PSObject.Properties.Match("continuationToken")) {
                        $continuationToken = $response.continuationToken
                    } else {
                        $continuationToken = $null
                    }
```

Reply:
I must disagree. The first block is more elegant, readable, and logical. Let's not overcomplicate the simplicity.